src/Raven.Client/Documents/Operations/ETL/Queue/AmazonSqsConnectionSettings.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.ETL.Queue;
+
+public sealed class AmazonSqsConnectionSettings
+{
+    public Basic Basic { get; set; }
+
+    public bool Passwordless { get; set; }
+
+    public bool UseEmulator { get; set; }
```

Comment:
This was supposed to be `internal`, right? It's for our own testing purposes only

Reply:
Yes, but I think you suggested that maybe we should leave it so developers can use it for testing with the emulator.

Reply:
No, let's not encourage devs to use some 3rd party emulators. It might have potential issues and having this option available will suggest that we support that.

I'd rather prefer devs to test with the usage of the real Amazon SQS service.